src/main.rs

```diff
@@ -84,6 +94,66 @@ fn main() {
         });
         std::process::exit(0);
     }
+    // -Z help: print information on all unstable FeatureFlags (-Z flag)
+    else if config.feature_flag_enabled(FeatureFlag::HELP) {
+        config
+            .log_info(|config| {
+                let header = Style::new()
+                    .bold()
+                    .fg_color(Some(Color::Ansi(AnsiColor::Cyan)));
+                let option = Style::new().bold();
+
+                writeln!(config.stderr(), "{header}Unstable options{header:#}\n")?;
+                writeln!(
+                    config.stderr(),
+                    "{header}{:<20}{header:#}Description",
+                    "-Z name",
+                )?;
+
+                for flag in FeatureFlag::ALL_FLAGS.iter().filter(|x| !x.stable) {
+                    write!(config.stderr(), "{option}{:<20}{option:#}", flag.id)?;
+
+                    if let Some(help) = flag.help {
+                        let mut lines = help.lines();
+
+                        if let Some(first) = lines.next() {
+                            writeln!(config.stderr(), "{first}")?;
+
+                            for line in lines {
+                                writeln!(config.stderr(), "{:<20}{line}", "")?;
+                            }
+                        }
+                    } else {
+                        writeln!(config.stderr())?;
+                    }
+                }
```

Comment:
What do you think about printing these to stdout instead of stderr?

It might be confusing to run a command and have your desired output (the help text) appear on stderr instead of stdout. But if cargo prints to stderr then maybe I'm thinking about this wrong.